src/port.js

```diff
@@ -33,6 +34,11 @@ export default class Port {
 	connect() {
 		this.isConnected = true;
 		this.debug('connected');
+		const me = this;
+		this.eventQueue.forEach((evt) => {
+			me.sendMessage(`evt.${evt.type}`, evt.args);
```

Comment:
Duplicating this is less than amazing. eventQueue could alternatively be an array of functions to call upon connection?

```js
if (!this.isConnected) {
  this.eventQueue.push(() => {
    me.sendMessage(`evt.${eventType}`, data.args);
  });
  return this;
}

return this.sendMessage(`evt.${eventType}`, data.args);
```